sdk/python/tests/compiler/testdata/any_sequencer.py

```diff
@@ -49,8 +48,7 @@ def any_sequence_pipeline(
         image="registry.access.redhat.com/ubi8/ubi-minimal",
         command=["/bin/bash", "-c"],
         arguments=["sleep 30"]
-    ).apply(after_any([task1, task2, task3]))
-
+    ).apply(after_any([task1, task2, task3], "any_test"))
 
```

Comment:
Thanks Gang, could you let me know what's the mean of "any_test" defined here?

Reply:
Any sequency task name?

Reply:
Yes, this is the name. See the context:
```
Because Any Sequencer is created implicitly, by applying after_any function to another op. There are two problems here:

**any-sequencers can get no name or other user-defined metadata**
each op needs its own any-sequencer, even if it's identical to another one
```